cmd/soci/main.go

```diff
@@ -74,11 +76,17 @@ func main() {
 		cli.DurationFlag{
 			Name:  "timeout",
 			Usage: "timeout for commands",
+			Value: 30 * time.Second,
```

Comment:
nit: do we want a timeout by default? I believe CTR does not (although it does have a dial timeout to containerd itself which we should inherit)

Reply:
I believe that I encountered situations where our zero timeout was causing operations to immediately abort. I will more thoroughly explore those and document them here.

Reply:
That seems to have been an artifact of the development process. This is not necessary and I have removed it.